### ISSUE TITLE
Missing input file '@curl//:lib/pipeline.h'

### ISSUE THREAD

**Issue by Arun-George-Zachariah** [tensorflow/tensorflow#31770]
**System information**
- OS Platform and Distribution: Ubuntu 18.04
- TensorFlow installed from (source or binary): source
- TensorFlow version: 2.0
- Python version: 3.7.3
- Bazel version (if compiling from source): 0.25.3
- GCC/Compiler version (if compiling from source): 7.4.0

**Problem Description**
While building the pip package from source, I encounter the below error.
```ERROR: missing input file '@curl//:lib/pipeline.h'
ERROR: /users/arung/tensorflow/tensorflow/python/BUILD:4855:1: //tensorflow/python:pywrap_tensorflow_internal_py_wrap: missing input file '@curl//:lib/pipeline.h'
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /users/arung/tensorflow/tensorflow/python/BUILD:4855:1 1 input file(s) do not exist
INFO: Elapsed time: 82.408s, Critical Path: 15.68s
INFO: 675 processes: 675 local.
FAILED: Build did NOT complete successfully
```

**Sequence of commands**
`./configure`
`bazel build --config=v2 //tensorflow/tools/pip_package:build_pip_package`

**Comment by wdirons** [comment]
This should have been fixed yesterday by this commit:
https://github.com/tensorflow/tensorflow/commit/961eaa691ac4602bfb0fd04a3d6a73217ae63f69

[My teammate was getting the same error and it is now fixed in the latest 2.0 code.]

**Comment by ymodak** [comment]
Closing this since its resolved in latest tf 2.0 nightly. Feel free to reopen if having problems. Thanks!
